hatchery/pods_test.go

```diff
@@ -36,6 +38,8 @@ func TestBuildPodFromJSON(t *testing.T) {
 }
 
 func TestBuildPodFromDockstore(t *testing.T) {
+	defer SetupTest()()
```

Comment:
Oh this is clever. I had to read up on defer, so this is calling the function and deferring the function that it returns to be executed after the test returns. I think we need 2 things to make this make sense to people in the future,

- change `SetupTest` to `SetupAndTeardownTest` 
- a comment/docstring-like explanation about this behavior and intended use in the function

Reply:
that being said, I do think this is a lot cleaner and prefer it to the specially named file. I'm not too worried about having to put it in every test, it's similar to having to put conftest fixtures in every function test signature in Python